doc/howto_release.md

```diff
@@ -59,12 +59,12 @@ Now check if configure still works.
 If yes, submit to git:
 
 ```bash
-git checkout -b config_sub_update_r78
+git checkout -b config_sub_update_r80
 git add config.guess config.sub configure
 git commit -m"config.guess + config.sub: updated from http://git.savannah.gnu.org/cgit/config.git/plain/" config.guess config.sub configure
```

Comment:
Not related to changes in this PR, but why are the files added again here in the `git commit`?

Reply:
Right, fixed in 3f994c8eec5ac42eaa8f7170ccb40cf358e7624c